pkgs/applications/window-managers/cage/default.nix

```diff
@@ -17,6 +17,11 @@ stdenv.mkDerivation rec {
     sha256 = "0ixl45g0m8b75gvbjm3gf5qg0yplspgs0xpm2619wn5sygc47sb1";
   };
 
+  patches = [
+    # To fix the build with wlroots 0.14.0:
+    ./wlroots-0_14.patch
```

Comment:
Is this patch somewhere upstream? If so, please `fetchpatch` instead of inlining it into nixpkgs.

Reply:
Unfortunately the PR is still unmerged so there's no stable source yet (AFAIK). I'd also like to fetch it instead.